sdk/azidentity/client_certificate_credential_test.go

```diff
@@ -154,8 +148,8 @@ func TestClientCertificateCredential_GetTokenNoPrivateKey(t *testing.T) {
 	srv, close := mock.NewServer()
 	defer close()
 	srv.AppendResponse(mock.WithBody([]byte(accessTokenRespSuccess)))
-	srvURL := srv.URL()
-	_, err := NewClientCertificateCredential(tenantID, clientID, "testdata/certificate_nokey.pem", nil, &TokenCredentialOptions{HTTPClient: srv, AuthorityHost: &srvURL})
+	srv.URL()
```

Comment:
I think the rest of this line needs to be removed

Reply:
Fixed